tools/shell-commands/src/main/java/org/apache/unomi/shell/migration/impl/MigrationTo121.java

```diff
@@ -32,10 +32,11 @@
 /**
  * @author dgaillard
  */
-public class MigrationTo130 implements Migration {
+public class MigrationTo121 implements Migration {
```

Comment:
There is no 1.2.1 released at Apache, I'm not sure this is a good idea to do this.

Reply:
Maybe we should name these after what they do ? MigrateToNewTags / MigrateIndexTemplates ?

Reply:
I know it has not been released here, but on the other hand it has been released somewhere. It also does not break any existing functionality - all 1.2.x scripts will be executed when migrating to 1.3.0 after releasing it here.

This permits us to automate the process a bit, otherwise we will need to specify in the migration manual which scripts to run and which not.